ci/test.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -eEuo pipefail
+
+echo "" && echo "=== Set ENV vars ===" && echo ""
+export MIX_ENV=test
+export CHAIN_ID=77
+export ETHEREUM_JSONRPC_CASE=EthereumJSONRPC.Case.Nethermind.Mox
+export ETHEREUM_JSONRPC_WEB_SOCKET_CASE=EthereumJSONRPC.WebSocket.Case.Mox
+
+echo "" && echo "=== Mix Hex ===" && echo ""
```

Comment:
What is MIX just for my own awareness

Reply:
Mix is a build tool used in elixir projects: https://hexdocs.pm/mix/1.15.5/Mix.html

I can't give you much details as I'm no expert on elixir (yet)